ui/app/components/mount-backend-form.hbs

```diff
@@ -31,22 +31,18 @@
     <form {{on "submit" (perform this.mountBackend)}}>
       <FormFieldGroups
         @model={{@mountModel}}
-        @groupName={{if (eq @mountCategory "secret") "formFieldGroups"}}
+        @groupName="formFieldGroups"
```

Comment:
the use of this component and groupName being `formFieldGroups` has always been strange to me 🙃

Reply:
Ya it's a bit misleading since it corresponds to whatever you named the property on the model (now form). I should have probably just went with the default which is `fieldGroups` but now I've been consistently naming it `formFieldGroups` 😞.